src/templates/base/2019/accessibility_statement.html

```diff
@@ -0,0 +1,50 @@
+{% extends "%s/2019/base.html" % lang %}
```

Comment:
Why does this extend the language-specific base as opposed to the 2019 base?

Won't this overwrite all language-specific blocks into English?

Reply:
This is so it picks up the language specific text. There’s a lot of common, language-specific-text in that needed by **all** templates.

`”%s/2019/base.html”` actually extends `”base/2019/base.html”` but overrides the base text with language-specific base text.

Then base Accessibility extends that and overrides with Accessibility-specific code.

Then the language-specific Accessibility extends that and overrides with language-specific Accessibility-specific text.

It sounds very convoluted, but it’s the best way I could see to get a language specific base and then extend it with page-specific code, and then use that as template for language-specific, page-specific code. Wow that’s a mouthful!